docs/installation/addons.md

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
Rename this file to components.md or packaged-components.md. Its confusing that the name of the file is not in the Title.